### DISCUSSION TITLE
Unexpected error rendering tickets detail page

### DISCUSSION THREAD

**Discussion by trainor1** [django-helpdesk/django-helpdesk#1097] [Q&A] [answered]
Hi All,

This is my first try at using Discussions rather than Issues.  I am hoping that it works.

I am still in the very early stages of trying to configure and test Django-helpdesk.  I have been able to create a queue and open several tickets.  Every time I create a new ticket, I get a rendering error on the tickets detail page.  The ticket is created.  It just won't render the page.  I get the same error if I click on the link for the ticket.  This goes to the tickets detail page as well.  And I get the same error.  I will include a PDF of the error page below.  I am wondering if I have all of the required packages installed in my virtual environment.  I did my best to follow the install documentation.

I am attaching a PDF of the error page below:

[TypeError at _helpdesk_tickets_3_.pdf](https://github.com/django-helpdesk/django-helpdesk/files/11749835/TypeError.at._helpdesk_tickets_3_.pdf)

Thanks for your help.

Best,

Kevin

**Comment by Benbb96** [comment]
This could be an issue with the version of the Markdown package. Could you try to upgrade (or maybe downgrade) it to see if this solves the problem ?

**Reply by trainor1** [reply]
Before I start changing versions of markdown, I want to confirm the preferred version of Django to use with django-helpdesk.

I am using markdown 3.4.3.  Also, based upon advice that I found in the Installation section of the documentation, I am using Django  3.2.5.  In my own projects, I am up to using Django 4.1 and getting ready to go to 4.2.  Could some of my problem be coming from using Django 3.2?  Please share your thoughts about the best version of Django to be using with django-helpdesk.

Thanks.

**Reply by gwasser** [reply]
I believe django-helpdesk should support everything from 3.2 up to 4.2. The idea is to make it possible to migrate between Django LTS releases. So you can use Django 3.2, or jump to 4.2, for a new install. If running an old install, you can migrate Django and switch over.

**Comment by Benbb96** [comment]
Oh I see what's wrong ! Your packages versions are OK. In fact, this error has been fixed on August 2022 (cf. PR #1038) but last version of django-helpdesk was released on July 2022 so I think it's time to release to PyPi (@gwasser or @uhurusurfa ?)

If you don't want to wait, you can install django-helpdesk against the main branch to get the latest features and you will also be able to run on Django 4.  
To do that, you can run : 

    pip uninstall django-helpdesk
    pip install git+https://github.com/django-helpdesk/django-helpdesk.git@main

**Reply by gwasser** [reply]
Definitely needed a new release a while ago. I have a little more free time right now so I can take the current `main` branch release that passes tests and call that the next release (0.5 or whatever).

**Reply by gwasser** [reply]
@trainor1 there is now a 0.5.0 release on PyPi that should include a bunch of bugfixes. Of course feel free to open new issues if you run into any.

**Comment by trainor1** [comment, marked as answer]
Thanks for the help. I appreciate the quick response from both of you.

 I believe that these answers address my current issues.  I will install the new version from PyPi and run with Django 4.2.
